package/tests/test_client.py

```diff
@@ -1,13 +1,63 @@
 import unittest
+import os
 from aistore.client.api import Client
+from aistore.client.msg import Bck
+from aistore.client.const import PROVIDER_AIS
 
 
-class TestStringMethods(unittest.TestCase):
+class TestBasicPutGet(unittest.TestCase):
```

Comment:
Maybe, we should name this `TestBasicOps` or something similar. I believe the idea here is to test even `create`, `list`, `delete`, `put` bucket/objects.

Reply:
Renamed